.happy/terraform/modules/batch/main.tf

```diff
@@ -119,70 +119,6 @@ resource aws_batch_job_definition dataset_metadata_update {
 })
 }
 
-resource aws_batch_job_definition reprocess_dataset_metadata {
-  # this was used to reprocess dataset metadata in place when an error was found after publishing cellxgene schema 4.0
-  # TODO: can be removed after 4.0 migration is complete
-  type = "container"
-  name = "dp-${var.deployment_stage}-${var.custom_stack_name}-reprocess-dataset-metadata"
-  container_properties = jsonencode({
-  "command": ["python3", "-m", "backend.layers.processing.reprocess_dataset_metadata"],
-  "jobRoleArn": "${var.batch_role_arn}",
-  "image": "${var.image}",
-  "memory": var.batch_container_memory_limit,
-  "environment": [
-    {
-      "name": "ARTIFACT_BUCKET",
-      "value": "${var.artifact_bucket}"
-    },
-    {
-      "name": "CELLXGENE_BUCKET",
-      "value": "${var.cellxgene_bucket}"
-    },
-    {
-      "name": "DATASETS_BUCKET",
-      "value": "${var.datasets_bucket}"
-    },
-    {
-      "name": "DEPLOYMENT_STAGE",
-      "value": "${var.deployment_stage}"
-    },
-    {
-      "name": "AWS_DEFAULT_REGION",
-      "value": "${data.aws_region.current.name}"
-    },
-    {
-      "name": "REMOTE_DEV_PREFIX",
-      "value": "${var.remote_dev_prefix}"
-    }
-  ],
-  "vcpus": 8,
-  "linuxParameters": {
-     "maxSwap": 800000,
-     "swappiness": 60
-  },
-    "retryStrategy": {
-    "attempts": 3,
-    "evaluateOnExit": [
-      {
-          "action": "RETRY",
-          "onReason": "Task failed to start"
-      },
-      {
-          "action": "EXIT",
-          "onReason": "*"
-      }
-    ]
-  },
-  "logConfiguration": {
-    "logDriver": "awslogs",
-    "options": {
-      "awslogs-group": "${aws_cloudwatch_log_group.cloud_watch_logs_group.id}",
-      "awslogs-region": "${data.aws_region.current.name}"
-    }
-  }
-})
-}
-
```

Comment:
@nayib-jose-gloria @Bento007 are we sure we want to remove this?

Reply:
I thought when we were quarterbacking this at least one of you was a strong proponent of building this solution in to keep around. @Bento007 don't you have a slice of code you were working on to do some aspect of the reprocessing? Can't remember where you ended up on that...